examples/consul-ami/consul.json

```diff
@@ -2,7 +2,9 @@
   "min_packer_version": "0.12.0",
   "variables": {
     "aws_region": "us-east-1",
-    "consul_version": "1.2.2",
+    "consul_version": "1.0.0",
```

Comment:
You dowgraded the default version?

Reply:
Whoops. Unintentional, will fix. Good catch 👍